src/test/java/showcase/neodymium/tests/iframe/SelenideSwitchToFrameTest.java

```diff
@@ -0,0 +1,51 @@
+package showcase.neodymium.tests.iframe;
+
+import static com.codeborne.selenide.Condition.matchText;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.switchTo;
+
+import org.junit.Test;
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.Selenide;
+
+import io.qameta.allure.Description;
+import io.qameta.allure.junit4.Tag;
+import showcase.neodymium.tests.AbstractTest;
+
+/**
+ * This show case explains with an example implementation how to test a web site with frames (iframe).<br>
+ * Selenide offers a straight forward approach to handle such frames and within Neodymium this is simply applied.<br>
+ * <b>REQUIRED CONFIGURATION</b> <i>config/dev-neodymium.properties</i>:
+ * <ul>
+ * <li>neodymium.webDriver.chrome.pathToDriverServer = <local path to chrome webdriver executable> # webdriver
```

Comment:
Please remove the required configuration and the running in mvn part since this is not explicitly needed for the test case.